utils/utils.go

```diff
@@ -105,3 +107,17 @@ func CreateAccAddressFromBech32(address string, bech32prefix string) (addr sdk.A
 
 	return sdk.AccAddress(bz), nil
 }
+
+// ComputeIBCDenom compute the ibc voucher denom associated to
+// the portID and channelID of the precompile given a token denomination.
```

Comment:
```suggestion
// the portID and channelID of the precompile given a token denomination.
```
```suggestion
// ComputeIBCDenom compute the ibc voucher denom associated to
// the portID, channelID, and the given a token denomination.
```

This function is agnostic to the caller and precompile.